docs/operating-eck/eck-permissions.asciidoc

```diff
@@ -0,0 +1,112 @@
+:page_id: eck-permissions
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+[id="{p}-{page_id}"]
+= ECK permissions
+
+Installing and running ECK, as well as using ECK-managed resources requires certain Kubernetes link:https://kubernetes.io/docs/reference/access-authn-authz/rbac[permissions]. Each of those permissions is listed below, together with a reason why it is needed.
+
+
+[float]
+[id="{p}-{page_id}-defaults"]
+== Installing CRDs
+
+Below is the permission required to install CRDs. CRDs (link:https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/[CustomResourceDefinitions]) are the only non-namespaced resources required to be installed.
```

Comment:
`ValidatingWebhookConfiguration` is also a non-namespaced resource.

Reply:
```suggestion
This permission is required to install CRDs. CRDs (link:https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/[CustomResourceDefinitions]) are the only non-namespaced resources required to be installed.
```

Reply:
> `ValidatingWebhookConfiguration` is also a non-namespaced resource.

Yes, but it's optional. I want to emphasise here that out of all resources that ECK has to have to run, this is the only one that is non-namespaced.

Reply:
> Yes, but it's optional. I want to emphasise here that out of all resources that ECK has to have to run, this is the only one that is non-namespaced.

I think I have been confused by the fact that the `ValidatingWebhookConfiguration` appears later in a list of "permissions required to install ECK operator in a Kubernetes cluster". Should we then make it explicit that it is not the case for the `ValidatingWebhookConfiguration` ?

Reply:
Not sure if I understand what exactly you are suggesting. `ValidatingWebhookConfiguration` has `yes` in the `Optional?` column which I think already achieves that. Did you mean something else?

Reply:
My bad, I guess I focused on the "permissions required" in the first sentence, while there is indeed an "optional" column.